Libs/Core/ctkCorePythonQtDecorators.h

```diff
@@ -260,6 +260,51 @@ public Q_SLOTS:
   {
     return td->JobUID;
   }
+    void setCreationDateTime(ctkJobDetail* td, QString creationDateTime)
```

Comment:
If these are public methods (accessible from Python) then we need to check if `td==nullptr` and fail gracefully (log error and return) instead of crashing the application.